.github/workflows/beta-release.yml

```diff
@@ -21,7 +21,25 @@ jobs:
         with:
           node-version: 16.x
 
-      - name: Install Dependencies
+      - name: Get current node version
+        id: node-version
+        run: |
+          node_version=$(node -v)
+          echo "node_version=$node_version" >> $GITHUB_OUTPUT
+
+        # Attempt to cache all the node_modules directories based on the node version and package lock.
+      - name: Cache node_modules
+        id: npm-cache
+        uses: actions/cache@v3
+        env:
+          cache-name: cache-node-modules-${{ steps.node-version.outputs.node_version }}
+        with:
+          key: ${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          path: '**/node_modules'
```

Comment:
Pretty sure you will want to avoid caching `pages-e2e/workspaces/**/node_modules`

Reply:
And possibly anything else in `.gitignore`

Reply:
Because they won't be installed in the monorepo's root. They'll be installed in workspaces which aren't part of the monorepo.

Anything else in `.gitignore` is inherently not part of your source code—it'll be generated—so you presumably don't want to cache it.

Reply:
🤦 sorry I misread your suggestion, I thought you meant not to cache node_modules in the fixtures, my bad sorry! 🙏 

Regarding the `.gitignore` yeah I get what you mean, exclude everything in the gitignore that can contain node_modules 👍 (I'll check it, but it's likely that it's just the workspaces you mentioned 🙂)

Reply:
I think it should be all right now, please have a look 🙂